tests/functional/test_confirm_email_address.py

```diff
@@ -1,59 +1,55 @@
 import pytest
 from flask import url_for
 
-email_depts = [('some.one@digital.cabinet-office.gov.uk',
-                'Government Digital Service')]
-
 
+@pytest.mark.usefixtures('live_server')
 class When_on_confirm_email_with_yes_selected_and_valid_email(object):
 
-    @pytest.mark.parametrize("email_address,department", email_depts)
     def it_goes_to_department_confirm_when_continue_clicked(
-            self, live_server, browser, email_address, department):
-        browser.visit(url_for('main.request_email_address', _external=True))
+            self, browser, submit_known_email_address):
```

Comment:
Think I'll just leave the code as it is set up so that I can move on, may try this at a later stage though